apps/theming-designer/src/components/SemanticSlots.tsx

```diff
@@ -96,6 +103,7 @@ export const SemanticSlots: React.StatelessComponent<ISemanticSlotsProps> = (pro
     'disabledBackground', // neutralLighter
     'disabledText', // neutralTertiary
     'disabledSubText', // neutralQuaternary
+    'disabledBodySubtext', // neutralTertiaryAlt
```

Comment:
disabledBodySubtext belongs under Defaults group I believe - make sure to bump # to 17 too